src/components/developer/profile/Header.jsx

```diff
@@ -1,5 +1,40 @@
 const Header = () => {
-  return <div className="h-1/6 bg-blue-500">Header</div>;
+  return (
+    <div className="h-1/6 w-full bg-fuchsia-950 flex flex-row justify-center items-center px-6 py-1">
```

Comment:
what color is bg-fuchsia-950

Reply:
Where do I find the original colors according the screenshot ?